researchstudio-sat/webofneeds
# Details/Use Case Parsing #2253

Refactoring of RDF-parsing in use-cases and detail-definitions. In particular, there's a generic traversal utility function now (`getInFromJsonLd`) and parsing utilities, that mind the data-types attached to literals/ leaves via `@type` and does generic parsing (in a defensive way).

Test:

1. Create needs using every possible detail (e.g. one custom, one transport, one rent-search and one rent-offer).
2. Check if all published details can be parsed and needs are displayed as usual in the "post-details" view.

Resolves #2253
